CMakeLists.txt

```diff
@@ -39,6 +39,11 @@ endmacro()
 option(TORCH_MLIR_ENABLE_MHLO "Add mhlo dialect" ON)
 if(TORCH_MLIR_ENABLE_MHLO)
   add_definitions(-DTORCH_MLIR_ENABLE_MHLO)
+  option(TORCH_MLIR_ENABLE_MHLO_TRUNC_DIMSIZE_TO_I32 
```

Comment:
Can you add a "rationale" comment here justifying this flag and why we need it?

Reply:
done